WordPress/src/main/java/org/wordpress/android/models/JetpackSettingsModel.java

```diff
@@ -0,0 +1,56 @@
+package org.wordpress.android.models;
+
+import android.content.ContentValues;
+import android.database.Cursor;
+
+public class JetpackSettingsModel {
+    public static final String ID_COLUMN_NAME = "id";
+    private static final String JP_MONITOR_ACTIVE_COLUMN_NAME = "monitorActive";
+    private static final String JP_MONITOR_EMAIL_NOTES_COLUMN_NAME = "jpEmailNotifications";
+    private static final String JP_MONITOR_WP_NOTES_COLUMN_NAME = "jpWpNotifications";
+
+    public static final String JP_SETTINGS_TABLE_NAME = "jp_site_settings";
+    public static final String CREATE_JP_SETTINGS_TABLE_SQL =
```

Comment:
I am not sure how I feel about this sql and the column/table names being inside the model. I think this is a direct result of having the Jetpack settings table inside the site settings table.